docs/deploy/README.md

```diff
@@ -67,11 +67,11 @@ we need a plugin to share the target secret with the `volcano-global` namespace.
 
 ```bash
 # Switch to Karmada host kubeconfig.
-export KUBECONFIG=/Users/vacant/.kube/karmada.config
+export KUBECONFIG=$HOME/.kube/karmada.config
 
-# Deploy the Kubernetes Reflector and share the kubeconfig from karmada-system namespace.
+# Deploy the Kubernetes Reflector and share the karmada-webhook-config secret from karmada-system namespace, it includes the kubeconfig of Karmada control plane.
 kubectl --context karmada-host -n kube-system apply -f https://github.com/emberstack/kubernetes-reflector/releases/download/v7.1.262/reflector.yaml
-kubectl --context karmada-host annotate secret kubeconfig \
+kubectl --context karmada-host annotate secret karmada-webhook-config \
```

Comment:
Karmada removes the kubeconfig secret, we must get the kubeconfig from the another secret